### ISSUE TITLE
Some options do not work, depending on the name of the frontend (Backend:file)

### ISSUE THREAD

**Issue by zorgzerg** [containous/traefik#3277]
### Do you want to request a *feature* or report a *bug*?

BUG

### What did you do?

- Created external backend named "backend-nsk0-web0" in separate toml file
- Created frontend named "new" in separate toml file

### What did you expect to see?

- Expect basic auth request at **http://*.new.domain.com** and redirect to **https://*.new.domain.com**

### What did you see instead?
- No basic auth
- No redirect

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.6.0
Codename:     tetedemoine
Go version:   go1.10.1
Built:        2018-04-30_09:28:44PM
OS/Arch:      linux/amd64
```

#### Backend config
```toml
[backends]
  [backends.backend-nsk0-web0]
    [backends.backend-nsk0-web0.LoadBalancer]
      method = "drr"
    [backends.backend-nsk0-web0.servers]
      [backends.backend-nsk0-web0.servers.external]
        url = "http://nsk0-web0.domain.com"
```

#### This is frontend worked not properly
```toml
[frontends]
  [frontends.new]
    entryPoints = ["http", "https"]
    backend = "backend-nsk0-web0"
    passHostHeader = true

    basicAuth = [
      "test:$apr1$H6uskkkW$IgXLP6ewTrSuBkTrqE8wj/",
      "test2:$apr1$d9hr9HBB$4HxwgUir3HP4EsggP/QNo0",
    ]

    [frontends.new.headers]
      SSLRedirect = true
      SSLTemporaryRedirect = true

    [frontends.new.routes]
      [frontends.new.routes.main]
        rule = "HostRegexp:{subdomain:[A-Za-z0-9-]+}.new.domain.com,new.domain.com"
```

#### But this is frontend worked fine
```toml
[frontends]
  [frontends.front-new]
    entryPoints = ["http", "https"]
    backend = "backend-nsk0-web0"
    passHostHeader = true

    basicAuth = [
      "test:$apr1$H6uskkkW$IgXLP6ewTrSuBkTrqE8wj/",
      "test2:$apr1$d9hr9HBB$4HxwgUir3HP4EsggP/QNo0",
    ]

    [frontends.front-new.headers]
      SSLRedirect = true
      SSLTemporaryRedirect = true

    [frontends.front-new.routes]
      [frontends.front-new.routes.main]
        rule = "HostRegexp:{subdomain:[A-Za-z0-9-]+}.new.domain.com,new.domain.com"
```

Also:
- new1 - bad
- 1new - ok
- frontend-new - bad
- frontend1-new - bad
- frontend-1new - ok
- 1frontend-new - ok
etc.

#### Debug log with "new" frontend
```
time="2018-05-07T03:12:26Z" level=debug msg="vulcand/oxy/roundrobin/rebalancer: begin ServeHttp on request" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"\",\"Opaque\":\"\",\"User\":null,\"Host\":\"\",\"Path\":\"/\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/1.1\",\"ProtoMajor\":1,\"ProtoMinor\":1,\"Header\":{\"Accept\":[\"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8\"],\"Accept-Encoding\":[\"gzip, deflate\"],\"Accept-Language\":[\"ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7\"],\"Connection\":[\"keep-alive\"],\"Cookie\":[\"_ga=GA1.2.1001060792.1523939493; _ym_uid=1523939493609979041\"],\"Upgrade-Insecure-Requests\":[\"1\"],\"User-Agent\":[\"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"new.domain.com\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"192.168.88.99:57820\",\"RequestURI\":\"/\",\"TLS\":null}"
time="2018-05-07T03:12:26Z" level=debug msg="vulcand/oxy/roundrobin/rebalancer: Forwarding this request to URL" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"\",\"Opaque\":\"\",\"User\":null,\"Host\":\"\",\"Path\":\"/\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/1.1\",\"ProtoMajor\":1,\"ProtoMinor\":1,\"Header\":{\"Accept\":[\"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8\"],\"Accept-Encoding\":[\"gzip, deflate\"],\"Accept-Language\":[\"ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7\"],\"Connection\":[\"keep-alive\"],\"Cookie\":[\"_ga=GA1.2.1001060792.1523939493; _ym_uid=1523939493609979041\"],\"Upgrade-Insecure-Requests\":[\"1\"],\"User-Agent\":[\"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"new.domain.com\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"192.168.88.99:57820\",\"RequestURI\":\"/\",\"TLS\":null}" ForwardURL="http://nsk0-web0.domain.com"
time="2018-05-07T03:12:26Z" level=debug msg="vulcand/oxy/forward: begin ServeHttp on request" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"http\",\"Opaque\":\"\",\"User\":null,\"Host\":\"nsk0-web0.domain.com\",\"Path\":\"\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/1.1\",\"ProtoMajor\":1,\"ProtoMinor\":1,\"Header\":{\"Accept\":[\"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8\"],\"Accept-Encoding\":[\"gzip, deflate\"],\"Accept-Language\":[\"ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7\"],\"Connection\":[\"keep-alive\"],\"Cookie\":[\"_ga=GA1.2.1001060792.1523939493; _ym_uid=1523939493609979041\"],\"Upgrade-Insecure-Requests\":[\"1\"],\"User-Agent\":[\"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"new.domain.com\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"192.168.88.99:57820\",\"RequestURI\":\"/\",\"TLS\":null}"
time="2018-05-07T03:12:26Z" level=debug msg="vulcand/oxy/forward/http: begin ServeHttp on request" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"http\",\"Opaque\":\"\",\"User\":null,\"Host\":\"nsk0-web0.domain.com\",\"Path\":\"\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/1.1\",\"ProtoMajor\":1,\"ProtoMinor\":1,\"Header\":{\"Accept\":[\"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8\"],\"Accept-Encoding\":[\"gzip, deflate\"],\"Accept-Language\":[\"ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7\"],\"Connection\":[\"keep-alive\"],\"Cookie\":[\"_ga=GA1.2.1001060792.1523939493; _ym_uid=1523939493609979041\"],\"Upgrade-Insecure-Requests\":[\"1\"],\"User-Agent\":[\"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"new.domain.com\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"192.168.88.99:57820\",\"RequestURI\":\"/\",\"TLS\":null}"
time="2018-05-07T03:12:26Z" level=debug msg="vulcand/oxy/forward/http: Round trip: http://nsk0-web0.domain.com, code: 502, Length: 575, duration: 5.915423ms"
time="2018-05-07T03:12:26Z" level=debug msg="vulcand/oxy/forward/http: completed ServeHttp on request" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"http\",\"Opaque\":\"\",\"User\":null,\"Host\":\"nsk0-web0.domain.com\",\"Path\":\"\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/1.1\",\"ProtoMajor\":1,\"ProtoMinor\":1,\"Header\":{\"Accept\":[\"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8\"],\"Accept-Encoding\":[\"gzip, deflate\"],\"Accept-Language\":[\"ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7\"],\"Connection\":[\"keep-alive\"],\"Cookie\":[\"_ga=GA1.2.1001060792.1523939493; _ym_uid=1523939493609979041\"],\"Upgrade-Insecure-Requests\":[\"1\"],\"User-Agent\":[\"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"new.domain.com\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"192.168.88.99:57820\",\"RequestURI\":\"/\",\"TLS\":null}"
time="2018-05-07T03:12:26Z" level=debug msg="vulcand/oxy/forward: completed ServeHttp on request" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"http\",\"Opaque\":\"\",\"User\":null,\"Host\":\"nsk0-web0.domain.com\",\"Path\":\"\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/1.1\",\"ProtoMajor\":1,\"ProtoMinor\":1,\"Header\":{\"Accept\":[\"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8\"],\"Accept-Encoding\":[\"gzip, deflate\"],\"Accept-Language\":[\"ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7\"],\"Connection\":[\"keep-alive\"],\"Cookie\":[\"_ga=GA1.2.1001060792.1523939493; _ym_uid=1523939493609979041\"],\"Upgrade-Insecure-Requests\":[\"1\"],\"User-Agent\":[\"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"new.domain.com\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"192.168.88.99:57820\",\"RequestURI\":\"/\",\"TLS\":null}"
time="2018-05-07T03:12:26Z" level=debug msg="vulcand/oxy/roundrobin/rebalancer: competed ServeHttp on request" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"\",\"Opaque\":\"\",\"User\":null,\"Host\":\"\",\"Path\":\"/\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/1.1\",\"ProtoMajor\":1,\"ProtoMinor\":1,\"Header\":{\"Accept\":[\"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8\"],\"Accept-Encoding\":[\"gzip, deflate\"],\"Accept-Language\":[\"ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7\"],\"Connection\":[\"keep-alive\"],\"Cookie\":[\"_ga=GA1.2.1001060792.1523939493; _ym_uid=1523939493609979041\"],\"Upgrade-Insecure-Requests\":[\"1\"],\"User-Agent\":[\"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"new.domain.com\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"192.168.88.99:57820\",\"RequestURI\":\"/\",\"TLS\":null}"
```
#### Debug log with "front-new" frontend

```
time="2018-05-07T03:16:36Z" level=debug msg="Basic auth failed"
time="2018-05-07T03:16:47Z" level=debug msg="Basic auth succeeded"
time="2018-05-07T03:16:47Z" level=debug msg="vulcand/oxy/roundrobin/rebalancer: begin ServeHttp on request" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"\",\"Opaque\":\"\",\"User\":{},\"Host\":\"\",\"Path\":\"/\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/2.0\",\"ProtoMajor\":2,\"ProtoMinor\":0,\"Header\":{\"Accept\":[\"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7\"],\"Authorization\":[\"Basic dGVzdDp0ZXN0\"],\"Cache-Control\":[\"max-age=0\"],\"Cookie\":[\"_ga=GA1.2.1001060792.1523939493; _ym_uid=1523939493609979041\"],\"Upgrade-Insecure-Requests\":[\"1\"],\"User-Agent\":[\"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"new.domain.com\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"192.168.88.99:54046\",\"RequestURI\":\"/\",\"TLS\":null}"
time="2018-05-07T03:16:47Z" level=debug msg="vulcand/oxy/roundrobin/rebalancer: Forwarding this request to URL" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"\",\"Opaque\":\"\",\"User\":{},\"Host\":\"\",\"Path\":\"/\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/2.0\",\"ProtoMajor\":2,\"ProtoMinor\":0,\"Header\":{\"Accept\":[\"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7\"],\"Authorization\":[\"Basic dGVzdDp0ZXN0\"],\"Cache-Control\":[\"max-age=0\"],\"Cookie\":[\"_ga=GA1.2.1001060792.1523939493; _ym_uid=1523939493609979041\"],\"Upgrade-Insecure-Requests\":[\"1\"],\"User-Agent\":[\"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"new.domain.com\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"192.168.88.99:54046\",\"RequestURI\":\"/\",\"TLS\":null}" ForwardURL="http://nsk0-web0.domain.com"
time="2018-05-07T03:16:47Z" level=debug msg="vulcand/oxy/forward: begin ServeHttp on request" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"http\",\"Opaque\":\"\",\"User\":null,\"Host\":\"nsk0-web0.domain.com\",\"Path\":\"\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/2.0\",\"ProtoMajor\":2,\"ProtoMinor\":0,\"Header\":{\"Accept\":[\"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7\"],\"Authorization\":[\"Basic dGVzdDp0ZXN0\"],\"Cache-Control\":[\"max-age=0\"],\"Cookie\":[\"_ga=GA1.2.1001060792.1523939493; _ym_uid=1523939493609979041\"],\"Upgrade-Insecure-Requests\":[\"1\"],\"User-Agent\":[\"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"new.domain.com\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"192.168.88.99:54046\",\"RequestURI\":\"/\",\"TLS\":null}"
time="2018-05-07T03:16:47Z" level=debug msg="vulcand/oxy/forward/http: begin ServeHttp on request" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"http\",\"Opaque\":\"\",\"User\":null,\"Host\":\"nsk0-web0.domain.com\",\"Path\":\"\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/2.0\",\"ProtoMajor\":2,\"ProtoMinor\":0,\"Header\":{\"Accept\":[\"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7\"],\"Authorization\":[\"Basic dGVzdDp0ZXN0\"],\"Cache-Control\":[\"max-age=0\"],\"Cookie\":[\"_ga=GA1.2.1001060792.1523939493; _ym_uid=1523939493609979041\"],\"Upgrade-Insecure-Requests\":[\"1\"],\"User-Agent\":[\"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"new.domain.com\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"192.168.88.99:54046\",\"RequestURI\":\"/\",\"TLS\":null}"
time="2018-05-07T03:16:47Z" level=debug msg="vulcand/oxy/forward/http: Round trip: http://nsk0-web0.domain.com, code: 502, Length: 575, duration: 6.04039ms tls:version: 303, tls:resume:false, tls:csuite:cca8, tls:server:new.domain.com"
time="2018-05-07T03:16:47Z" level=debug msg="vulcand/oxy/forward/http: completed ServeHttp on request" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"http\",\"Opaque\":\"\",\"User\":null,\"Host\":\"nsk0-web0.domain.com\",\"Path\":\"\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/2.0\",\"ProtoMajor\":2,\"ProtoMinor\":0,\"Header\":{\"Accept\":[\"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7\"],\"Authorization\":[\"Basic dGVzdDp0ZXN0\"],\"Cache-Control\":[\"max-age=0\"],\"Cookie\":[\"_ga=GA1.2.1001060792.1523939493; _ym_uid=1523939493609979041\"],\"Upgrade-Insecure-Requests\":[\"1\"],\"User-Agent\":[\"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"new.domain.com\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"192.168.88.99:54046\",\"RequestURI\":\"/\",\"TLS\":null}"
time="2018-05-07T03:16:47Z" level=debug msg="vulcand/oxy/forward: completed ServeHttp on request" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"http\",\"Opaque\":\"\",\"User\":null,\"Host\":\"nsk0-web0.domain.com\",\"Path\":\"\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/2.0\",\"ProtoMajor\":2,\"ProtoMinor\":0,\"Header\":{\"Accept\":[\"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7\"],\"Authorization\":[\"Basic dGVzdDp0ZXN0\"],\"Cache-Control\":[\"max-age=0\"],\"Cookie\":[\"_ga=GA1.2.1001060792.1523939493; _ym_uid=1523939493609979041\"],\"Upgrade-Insecure-Requests\":[\"1\"],\"User-Agent\":[\"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"new.domain.com\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"192.168.88.99:54046\",\"RequestURI\":\"/\",\"TLS\":null}"
time="2018-05-07T03:16:47Z" level=debug msg="vulcand/oxy/roundrobin/rebalancer: competed ServeHttp on request" Request="{\"Method\":\"GET\",\"URL\":{\"Scheme\":\"\",\"Opaque\":\"\",\"User\":{},\"Host\":\"\",\"Path\":\"/\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"},\"Proto\":\"HTTP/2.0\",\"ProtoMajor\":2,\"ProtoMinor\":0,\"Header\":{\"Accept\":[\"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Accept-Language\":[\"ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7\"],\"Authorization\":[\"Basic dGVzdDp0ZXN0\"],\"Cache-Control\":[\"max-age=0\"],\"Cookie\":[\"_ga=GA1.2.1001060792.1523939493; _ym_uid=1523939493609979041\"],\"Upgrade-Insecure-Requests\":[\"1\"],\"User-Agent\":[\"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36\"]},\"ContentLength\":0,\"TransferEncoding\":null,\"Host\":\"new.domain.com\",\"Form\":null,\"PostForm\":null,\"MultipartForm\":null,\"Trailer\":null,\"RemoteAddr\":\"192.168.88.99:54046\",\"RequestURI\":\"/\",\"TLS\":null}"
```

**Comment by arnested** [comment]
Sounds like a duplicate of #2323.

**Comment by zorgzerg** [comment]
Guys, it will be fix in 1.7?

**Comment by arnested** [comment]
If it _is_ a duplicate of #2323 it is fixed in 1.7 (#3312).
